frontend/src/app/modules/authentication/registration/registration.component.ts

```diff
@@ -9,29 +18,59 @@ import { UserRegisterDto } from '../../../models/auth/user-register-dto';
     templateUrl: './registration.component.html',
     styleUrls: ['./registration.component.sass'],
 })
-export class RegistrationComponent implements OnInit {
+export class RegistrationComponent extends BaseComponent implements OnInit {
     public registerForm: FormGroup = new FormGroup({});
 
-    // eslint-disable-next-line no-empty-function
-    constructor(private fb: FormBuilder) {
+    public showPassword = false;
+
+    public showConfirmPassword = false;
```

Comment:
what are you using these parameters for?

Reply:
hm, i didn't add these things, maybe they left from previous commits

Reply:
i will remove them, seems like they don't change anything